test_tableslurp.sh

```diff
@@ -109,44 +110,50 @@ echo
 
 # Run tableslurp with current & extra 4 args. Then do a check.
 # --name tester - is the flag & the replacement container ID, "${CONTAINER}"
-# index_file is: index.txt
-# index_bucket is: kimlew-tablesnap-tests
+# index_file is:    index.txt
+# index_bucket is:  kimlew-tablesnap-tests
 # backup_bucket is: kimlew-tablesnap-tests
 
-# Rename the local file with mv to same directory. Check removal. Then re-run
-# tableslurp & check if file exists to prove file was retrieved from AWS.
-echo "${SRC_LOCAL}"
-echo "${ORIG_TEST_FILE_LOC}"
-mv "${SRC_LOCAL}" "${ORIG_TEST_FILE_LOC}"
+# Rename the local file with mv, to same directory as prep for download from AWS.
+# Check removal. Run tableslurp. Check if files are the same locally, to prove
+# successful file download.
+echo "SRC_LOCAL_UPDOWN_FILE:\t" "${SRC_LOCAL_UPDOWN_FILE}"
+echo "SRC_LOCAL_ORIG_TEST_FILE:\t" "${SRC_LOCAL_ORIG_TEST_FILE}"
+mv "${SRC_LOCAL_UPDOWN_FILE}" "${SRC_LOCAL_ORIG_TEST_FILE}"
+echo
 
 # Check if test file is temporarily removed from /tmp/tablesnap-test.
-if [ -f "${SRC_LOCAL}" ]; then
+if [ -f "${SRC_LOCAL_UPDOWN_FILE}" ]; then
   echo "${UPDOWN_FILE} was NOT deleted. Directory has:"
-  ls "${DIR_LOCAL}"
+  ls -lah "${DIR_LOCAL}"
 else
   echo "${UPDOWN_FILE} has been deleted. Directory now has:"
-  ls "${DIR_LOCAL}"
+  ls -lah "${DIR_LOCAL}"
```

Comment:
I suggest calling `die` with a message here. If you can't `mv` the file, there is something wrong with the test.

Reply:
Looks good.